build/FLExBridge.proj

```diff
@@ -14,9 +14,9 @@
 		<ExtraExcludeCategories Condition="'$(OS)'!='Windows_NT'">KnownMonoIssue,UnknownMonoIssue</ExtraExcludeCategories>
 		<ExtraExcludeCategories Condition="'$(teamcity_version)' != ''">SkipOnTeamCity,$(ExtraExcludeCategories)</ExtraExcludeCategories>
 		<Release Condition="'$(Release)' == ''">true</Release>
-		<GitVersionTaskAltPath>$(RootDir)/packages/GitVersionTask.alt/build</GitVersionTaskAltPath>
-		<RestartBuild Condition="!Exists('$(GitVersionTaskAltPath)/GitVersionTask.dll') Or !Exists('$(RootDir)/packages/SIL.BuildTasks/tools/SIL.BuildTasks.dll') Or !Exists('$(RootDir)/packages/SIL.ReleaseTasks/build/net461/SIL.ReleaseTasks.props')">true</RestartBuild>
-		<RestartBuild Condition="Exists('$(GitVersionTaskAltPath)/GitVersionTask.dll') And Exists('$(RootDir)/packages/SIL.BuildTasks/tools/SIL.BuildTasks.dll') And Exists('$(RootDir)/packages/SIL.ReleaseTasks/build/net461/SIL.ReleaseTasks.props')">false</RestartBuild>
+		<GitVersionTaskPath>$(RootDir)/packages/GitVersion.MsBuild.5.6.8/build</GitVersionTaskPath>
```

Comment:
One improvement would be to put the version number in a property and use that, here and below.